pkg/yurtmanager/webhook/nodepool/v1beta1/nodepool_default.go

```diff
@@ -38,6 +39,14 @@ func (webhook *NodePoolHandler) Default(ctx context.Context, obj runtime.Object)
 		np.Spec.Type = v1beta1.Edge
 	}
 
+	if np.Labels == nil {
+		np.Labels = map[string]string{
+			apps.NodePoolTypeLabel: string(np.Spec.Type),
```

Comment:
how about use low letters only? this means:

`string(np.Spec.Type) --> strings.ToLower(string(np.Spec.Type))`

Reply:
done